widgets/views/versionCalendar.php

```diff
@@ -17,7 +17,7 @@
 }
 if (!isset($non_standalone)) {
 	header('Content-Type: image/svg+xml');
-	echo '<?xml version="1.0"?>';
+	echo '<?xml version="1.0" encoding="UTF-8"?>';
```

Comment:
That won't help passing validation. `$non_standalone` seems not to be passed properly since SVG is used inline but these headers are sent.